src/Model/Item.php

```diff
@@ -263,6 +263,7 @@ public static function insert($arr, $force_parent = false, $notify = false, $don
 		}
 
 		$arr['guid'] = notags(trim(defaults($arr, 'guid', get_guid(32, $guid_prefix))));
+		/// @todo Declare $uid variable - Rabuzarus - 2018-02-12.
```

Comment:
Yep, it should be `$arr['uid']`.

Reply:
Or line 306 should be moved some lines higher.